src/components/App.js

```diff
@@ -1,10 +1,35 @@
 import React from 'react'
-import '../App.css'
+import BookList from '../containers/BookList'
+import BookForm from '../containers/BookForm'
+import Navbar from '../components/Navbar'
+import Loader from './loader'
+import { fetchBooks } from '../actions'
+import { connect } from 'react-redux'
 
-function App () {
-  return (
-    <h1> Hello BookStore</h1>
-  )
+class App extends React.Component {
+  componentDidMount () {
+    this.props.fetchBooks()
+  }
+
+  render () {
+    return (
+      <div>
+        <Navbar />
+        <div className='container'>
+          <Loader>
+            <BookList />
+            <BookForm />
+          </Loader>
+        </div>
+      </div>
+    )
+  }
+}
+
+function mapDispatchToProps (dispatch) {
```

Comment:
For this and all the other dispatch actions we are using in all the other containers, we can shortcut our call to connect without having to declare a `mapDispatchToProps`:

```javascript
export default connect(null, { fetchBooks })(App)
```